src/main.rs

```diff
@@ -94,14 +96,25 @@ async fn main() {
 
                 path.to_string_lossy().to_string()
             } else {
-                uri.path().to_string()
+                let re = Regex::new(r"screenshot-(\d{4}-\d{2}-\d{2}-\d{2}-\d{2}-\d{2})").unwrap();
```

Comment:
It is not correct to try and replace this path with a regex, instead the path should be adjusted where it is originally created.